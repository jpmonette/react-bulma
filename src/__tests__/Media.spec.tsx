import * as React from "react";
import { create } from "react-test-renderer";

import { Media } from "../";

describe("Level", () => {
  it("should generate a level section", () => {
    const tree = create(
      <Media>
        <Media.Left className="sample-class">
          <Media.Content>Media Content</Media.Content>
        </Media.Left>
        <Media.Right className="sample-class">
          <Media.Content>Media Content</Media.Content>
          <Media.Content>Media Content</Media.Content>
        </Media.Right>
      </Media>,
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
