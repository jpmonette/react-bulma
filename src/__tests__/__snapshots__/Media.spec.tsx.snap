// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Level should generate a level section 1`] = `
<article
  className="media"
>
  <div
    className="sample-class media-left"
  >
    <div
      className="media-content"
    >
      Media Content
    </div>
  </div>
  <div
    className="sample-class media-right"
  >
    <div
      className="media-content"
    >
      Media Content
    </div>
    <div
      className="media-content"
    >
      Media Content
    </div>
  </div>
</article>
`;
